opal/mca/reachable/netlink/configure.m4

```diff
@@ -159,16 +121,7 @@ AC_DEFUN([MCA_opal_reachable_netlink_CONFIG],[
 ])
 
     AS_IF([test $opal_reachable_netlink_happy -eq 1],
-          [OPAL_REACHABLE_NETLINK_CHECK_LIBNL3(
-                  [opal_reachable_netlink_LIBNL_CPPFLAGS],
-                  [opal_reachable_netlink_LIBNL_LIBS],
-                  [0])
-          ])
-    AS_IF([test "$opal_reachable_netlink_LIBNL_LIBS" == ""],
-          [opal_reachable_netlink_happy=0])
-
-    AC_SUBST(opal_reachable_netlink_LIBNL_CPPFLAGS)
-    AC_SUBST(opal_reachable_netlink_LIBNL_LIBS)
+          [OPAL_REACHABLE_NETLINK_CHECK_LIBNL_xxx])
```

Comment:
I realize why you used the `_xxx` suffix (to distinguish that it's not "libnl"), but it's still a bit weird...